storybook/src/pages/amsterdam/home/HomeNews.tsx

```diff
@@ -0,0 +1,45 @@
+import { AspectRatio, Card, Grid, Heading, Image } from '@amsterdam/design-system-react'
+
+export const HomeNews = () => (
```

Comment:
Also would opt for the re-use of the card story.

Reply:
We don’t have a Story in this shape. Our example Headings aren’t very newsy. And it’s a bit laborious to ensure the three headings are different. I would say this is fine.